src/components/Editor/Runners/HtmlRunner/HtmlRunner.jsx

```diff
@@ -168,70 +209,88 @@ function HtmlRunner() {
   }, [runningFile]);
 
   const runCode = () => {
-    setRunningFile(previewFile);
```

Comment:
I don't really understand why this was deleted?

Reply:
Hmmm must be an error on my part. I've reinstated